packages/host/app/lib/matrix-classes/message-command.ts

```diff
@@ -73,6 +73,21 @@ export default class MessageCommand {
     }
   }
 
+  async getCommandCodeRef() {
+    let roomResource = this.matrixService.roomResources.get(
+      this.message.roomId,
+    );
+    if (!roomResource) {
+      return undefined;
+    }
+    // We need to wait for the skills to actually be loaded
+    await roomResource.waitForAllSkills();
```

Comment:
Without this, the first time you load the page, they don't work, clicking again works fine.